index.js

```diff
@@ -30,6 +30,7 @@ Cypress.Commands.add('upload', { prevSubject: 'element' }, (subject, fileOrArray
       processedFiles.forEach(file => dataTransfer.items.add(file));
       const input = subject[0];
       input.files = dataTransfer.files;
+      return cy.wrap(subject).trigger('change');
```

Comment:
Does that means for Chrome <73, Electron (& likely other browsers as well) the `change` event will occur twice?

Reply:
@abramenal I will check today, but there were no problems on my code base.

Reply:
That is important since in case someone who's code relies on `change` event handling will get twice more executions even if it uses Electron – that will be a big issue. Let's doublecheck that 😃

Reply:
@abramenal test in electron, event fire twice  :(
![image](https://user-images.githubusercontent.com/1303845/54666853-7a547180-4b1d-11e9-98e7-de8869c028d1.png)

Reply:
Solutions:
- add a description of this behavior in the readme and do not change the code
- change the code and add a description, works with the Chrome version> 73, in the old versions there will be 2 events
- add checks on the chrome version to the code

Reply:
well, I'll create a check for this behavior in chrome> 73.

it would also be nice to find a description of the problem, now there is no place anywhere ((

Reply:
You mean Chrome’s changelog? Or?

Reply:
I just encounter this issue on my test. I would suggest use `trigger('change', { force: true })` to make it work when input is hidden which is common case.
And I vote for write a tricky code to detect browser version.